sem_covid/entrypoints/notebooks/EDA/eda_wrangling/word_handling.py

```diff
@@ -104,10 +104,11 @@ def get_words(data: pd.Series) -> pd.Series:
 
 def delete_punctuation(text: str) -> str:
     """
-    Function to delete punctuation form text
+    Function to delete punctuation from text
```

Comment:
This shall be marked for deprecation (90% sure there is another one somewhere doing this better)